azure-quantum/azure/quantum/optimization/solvers.py

```diff
@@ -16,14 +16,34 @@
 logger = logging.getLogger(__name__)
 
 __all__ = [
+    'RangeSchedule',
     'Solver',
     'ParallelTempering',
     'SimulatedAnnealing',
     'HardwarePlatform',
     'Tabu',
-    'QuantumMonteCarlo'
+    'QuantumMonteCarlo',
+    'PopulationAnnealing',
+    'SubstochasticMonteCarlo'
 ]
 
+class RangeSchedule: 
+    def __init__(self, type: str, initial: float, final: float): 
```

Comment:
```suggestion
    def __init__(self, range_type: str, initial: float, final: float): 
```

Reply:
the type is for the whole class obj, range_type will be confusing.

Reply:
what about `scheduler_type` or something else less generic than just `type`?

Reply:
I've seen people use `_type` also.